### PR TITLE
Update State Implementation

### PR DESCRIPTION
**Pull request by nrwslac**
- change coating state PV names to `MR*L*:HOMS:COATING:STATE` removing axis designation in the PV
- move state code to its own program `PRG_States` for readability
- cleaned up some spacing.

Untested, but I don't see a reason why this state code would perform any differently out of a PRG block; will test and deploy at next down time.

### CODE REVIEW

**Review by ZLLentz — APPROVED** [review]
Seems reasonable. I didn't check this line-by-line because of the tedium. It might be a good idea to rebuild the IOC to make sure that the PVs didn't change in unexpected ways.

File: HOMS_XRT/HOMS_XRT_PLC/POUs/MAIN.TcPOU [hunk 1]
```diff
@@ -12,46 +12,18 @@ VAR
 	{attribute 'pytmc' := '
 		pv: MR1L3:HOMS:MMS:YUP
 	'}
-	M1 : DUT_MotionStage := (sName:='MR1L3-Coatings', fVelocity:=200.0, nEnableMode:=ENUM_StageEnableMode.ALWAYS, bPowerSelf:=TRUE);
+    //PMPS State Stage; bPowerSelf:=False
```
Comment (ghalym) [inline HOMS_XRT/HOMS_XRT_PLC/POUs/MAIN.TcPOU:15]: I am confused by this comment. Is it a requirement for all PMPS stages to have the bPowerSelf set to false?

Comment (nrwslac) [reply HOMS_XRT/HOMS_XRT_PLC/POUs/MAIN.TcPOU:15]: I thought this was the case. Have you seen otherwise?

Comment (ZLLentz) [reply HOMS_XRT/HOMS_XRT_PLC/POUs/MAIN.TcPOU:15]: If you want PMPS to have control over your enable/limits, you need this set to False.

Comment (ghalym) [reply HOMS_XRT/HOMS_XRT_PLC/POUs/MAIN.TcPOU:15]: I need to double check my projects then :-). Thanks Zach.

Comment (ZLLentz) [reply HOMS_XRT/HOMS_XRT_PLC/POUs/MAIN.TcPOU:15]: As-implemented, it should work out OK so long as PMPS is executed last

**Review by klauer — APPROVED** [review]
As best I can tell, the changes reflect your intentions and don't unintentionally modify any of the positions, assertion IDs, and other settings.

I would like to see that the IOC rebuild does result in the correct record database - that's something you'll pick up on right away during testing, though.

This looks good to me 👍

**Comment by nrwslac** [comment]
This was deployed and tested successfully. I'll merge a make release v4.3.0. Thanks all!

**Pull request merged**